src/main/kotlin/io/github/mojira/arisa/infrastructure/Jira.kt

```diff
@@ -54,6 +54,14 @@ fun resolveAsInvalid(issue: Issue) = runBlocking {
     }
 }
 
+fun resolveAsIncomplete(issue: Issue) = runBlocking {
+    Either.catch {
+        issue.transition()
+            .field(Field.RESOLUTION, "Incomplete")
+            .execute("Resolve Issue")
+    }
+}
```

Comment:
Maybe it would make sense to merge resolveAs methods to one with a string argument and pre-fill that argument using `partially`?

Reply:
Good idea

Reply:
Done